lib/Template/nodes/TemplateNodeComponent.js

```diff
@@ -78,6 +115,80 @@ export class TemplateNodeComponent extends TemplateNode {
     }
 
     /**
+     * @param {{}} pragma
+     * @returns {string}
+     * @private
+     */
+    _compileAsync(pragma) {
+        let async = pragma.async;
```

Comment:
variable name `async` will be conflict with reserved word (async/await). can we use `asyncMethod` instead it?